### ISSUE TITLE
Option to create an Invenio instance without user input (e.g. for use within CI)

### ISSUE THREAD

**Issue by apirogov** [inveniosoftware/invenio-cli#280]
I want to run blackbox tests against the backend API of Invenio RDM and would therefore like to have a possibility to create in instance within the CI pipeline to test against.

Currently, I am using an ugly workaround of creating instance using cookiecutter directly with `--no-input` and copying the `.invenio` file from a "validly" generated instance. This, obviously, is not a proper solution (and only works easily for the default values).

## Describe the solution you'd like
`invenio-cli init` should have a flag `--cookiecutter-config` that takes a JSON file with the values. It could be loaded and passed as a dict through the cookiecutter API as described [here](https://cookiecutter.readthedocs.io/en/1.7.0/advanced/suppressing_prompts.html).

## Additional context
If there is no opposition against this idea, I could also take care of implementing this little addition in the way outlined above. But first, I wanted to get other opinions.

**Comment by lnielsen** [comment]
Fully agree that it would be nice to provide the JSON config directly. 

I think both the options of ``--no-input`` and a ``--config`` would be nice. This way you can also quickly do an RDM template without having to go through the questions.

```invenio-cli init rdm --no-input --config=``

Also, thanks for offering to implement it. If this is the first time contributing just do a quick check of https://invenio.readthedocs.io/en/latest/community/contributing/maintainers-guide/copyright-policy.html

In a nutshell, you need to add copyright statements in the code, and the copyright should be assigned to a legal entity which you'll have to figure out who is. Could be you personally, or could be your employer (if you're employed), or if you're doing contract work, a contract might govern who gets copyright on the work you produce. After that, it's just getting a green light on CI system :-)

**Comment by apirogov** [comment]
Just as an update - I implemented the flags, now just waiting for confirmation about whether the Copyright notice is correct, and as soon as I get a "go" I'll open a pull request :)

Feel free to take a look in the meantime and tell me if it is ok.
